src/Kros.AspNetCore/Authorization/GatewayAuthorizationMiddleware.cs

```diff
@@ -89,7 +89,8 @@ internal class GatewayAuthorizationMiddleware
             {
                 client.DefaultRequestHeaders.Add(HeaderNames.Authorization, value.ToString());
 
-                HttpResponseMessage response = await client.GetAsync(_jwtAuthorizationOptions.AuthorizationUrl + httpContext.Request.Path.Value);
```

Comment:
Ja som pôvodne myslel, že bude extension priamo nad clientom. Niečo v zmysle `GetAndCheckResponseAsync()` a ona skontroluje response a vyhodí rovno exception. Aby programátor nemusel potom písať už ani ten `if`. @satano  čo ty na to?

Reply:
Dohodli sme sa, že ostane extension metóda nad `HttpResponseMessage` (napríklad `ThrowIfNotSuccessStatusCode()`), ktorá rovno spraví ten `if`. A zároveň sa spravia naše extension metódy nad klientom, ktoré budú používať tamto (`GetStringAndCheckResponse` - tu mám trochu problém s menom).

Reply:
Jop s tým menom mám problém aj ja :-) Dal som to len ako príklad, pretože nič lepšie mi nenapadlo.

Reply:
hotovo :)